Userland/Libraries/LibWebView/Options.h

```diff
@@ -50,6 +55,7 @@ struct ChromeOptions {
     AllowPopups allow_popups { AllowPopups::No };
     DisableScripting disable_scripting { DisableScripting::No };
     DisableSQLDatabase disable_sql_database { DisableSQLDatabase::No };
+    EnableAutoplay enable_autoplay { EnableAutoplay::No };
```

Comment:
I think this belongs in `WebContentOptions` - it controls the web view rather than the chrome process

Reply:
I've moved this. There are other options in `ChromeOptions` that should probably be moved as well if that's what the distinction is supposed to be.